tests/test_dirhash.py

```diff
@@ -132,7 +132,7 @@ def test_ignore_extensions(self):
 
 class TempDirTest(object):
 
-    def setup(self):
+    def setup_method(self):
         self.dir = tempfile.mkdtemp()
 
     def tear_down(self):
```

Comment:
looks like this should also be `teardown_method` :) https://docs.pytest.org/en/stable/how-to/xunit_setup.html